tools/topology/topology2/include/components/src_format_sxx_to_s32_convert.conf

```diff
@@ -2,345 +2,56 @@
 			num_input_audio_formats 42
 
 			Object.Base.input_audio_format [
-				# 8khz input
```

Comment:
and please also fix the prefix in the commit message to "topology2:..."

Reply:
@bitan the prefix is meant as an indication of the path to the file that is being changed.Can you please remove audio and expand to Topology2 ? POlease look at the log for other patches to see the right prefix